### PR TITLE
added sketch file

### LINKED ISSUE #4: proposal: sketch work
@jlee2843 will figure out the sketch based on correct data set, please feel free to @ us if you need some help with this one.

### PR DESCRIPTION
**Pull request by jlee2843**
Created a sketch of our web app. Details have been numbered on the sketch and explanations will be added on the proposal file later. I'll try to do this last to avoid any merge conflicts from arising.

### CODE REVIEW

**Comment by Kierst01** [comment]
I think we could reword this to make it clearer.
Under the #### Prediction Page section we could say:

"The second page, named Prediction, will feature the implementation of a simple machine learning model to predict the expected price per night based on the provided feature values."

Otherwise everything looks great!

**Comment by jlee2843** [comment]
> I think we could reword this to make it clearer. Under the #### Prediction Page section we could say:
> 
> "The second page, named Prediction, will feature the implementation of a simple machine learning model to predict the expected price per night based on the provided feature values."
> 
> Otherwise everything looks great!

Addressed! Thank you :)

**Comment by wenyunie** [comment]
@jlee2843 @Kierst01 Thank you for detailed change of this PR. It seems good to me now so I will just merge it~
